src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -20,7 +20,6 @@ import theme, { OakColorName } from "@/styles/theme";
 import errorReporter from "@/common-lib/error-reporter";
 import { VideoLocationValueType } from "@/browser-lib/avo/Avo";
 import OakError from "@/errors/OakError";
-import BoxBorders from "@/components/SharedComponents/SpriteSheet/BrushSvgs/BoxBorders/BoxBorders";
```

Comment:
These didn't match the player border styles and were not always loaded in time to be rendered for the pupil video page. I figure that a simple black border is enough here.

Reply:
Don't even get me started on boxBorders